Cannot compile MediaElch: "error: no matching function for call to ‘mediaelch::Locale::Locale()’"

**Describe the bug**
I cannot compile MediElch anymore. I have also tried to get whole new source files to fix, but I always end up in the same errors:

```
In file included from /usr/include/x86_64-linux-gnu/qt5/QtCore/qdebug.h:51,
                 from /usr/include/x86_64-linux-gnu/qt5/QtCore/QDebug:1,
                 from ../src/globals/Globals.h:6,
                 from ../src/scrapers/movie/MovieScraperInterface.h:4,
                 from ../src/scrapers/movie/AEBN.h:3,
                 from ../src/scrapers/movie/AEBN.cpp:1:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h: In instantiation of ‘void QVector<T>::reallocData(int, int, QArrayData::AllocationOptions) [with T = mediaelch::Locale; QArrayData::AllocationOptions = QFlags<QArrayData::AllocationOption>]’:
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h:390:13:   required from ‘void QVector<T>::detach() [with T = mediaelch::Locale]’
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h:205:31:   required from ‘QVector<T>::iterator QVector<T>::begin() [with T = mediaelch::Locale; QVector<T>::iterator = mediaelch::Locale*]’
../src/scrapers/movie/AEBN.cpp:35:67:   required from here
/usr/include/x86_64-linux-gnu/qt5/QtCore/qvector.h:607:33: error: no matching function for call to ‘mediaelch::Locale::Locale()’
  607 |                                 new (dst++) T();
      |                                 ^~~~~~~~~~~~~~~
In file included from ../src/scrapers/movie/MovieScraperInterface.h:3,
                 from ../src/scrapers/movie/AEBN.h:3,
                 from ../src/scrapers/movie/AEBN.cpp:1:
../src/data/Locale.h:24:5: note: candidate: ‘mediaelch::Locale::Locale(QString, QString)’
   24 |     Locale(QString language, QString country) : m_lang{language}, m_country{country} {}
      |     ^~~~~~
../src/data/Locale.h:24:5: note:   candidate expects 2 arguments, 0 provided
../src/data/Locale.h:22:18: note: candidate: ‘mediaelch::Locale::Locale(const QString&)’
   22 |     /*implicit*/ Locale(const QString& locale);
      |                  ^~~~~~
../src/data/Locale.h:22:18: note:   candidate expects 1 argument, 0 provided
../src/data/Locale.h:21:18: note: candidate: ‘mediaelch::Locale::Locale(const char*)’
   21 |     /*implicit*/ Locale(const char* locale) : Locale(QString(locale)) {}
      |                  ^~~~~~
../src/data/Locale.h:21:18: note:   candidate expects 1 argument, 0 provided
../src/data/Locale.h:12:7: note: candidate: ‘mediaelch::Locale::Locale(const mediaelch::Locale&)’
   12 | class Locale
      |       ^~~~~~
../src/data/Locale.h:12:7: note:   candidate expects 1 argument, 0 provided
../src/data/Locale.h:12:7: note: candidate: ‘mediaelch::Locale::Locale(mediaelch::Locale&&)’
../src/data/Locale.h:12:7: note:   candidate expects 1 argument, 0 provided
make: *** [Makefile:13028: AEBN.o] Error 1
make: *** Waiting for unfinished jobs....

```
**To Reproduce**
Steps to reproduce the behavior:
1. git clone https://github.com/Komet/MediaElch.git
2. cd MediaElch
3. git submodule update --init
4. mkdir build
5. cd build
6. qmake ..
7. make -j8

**Expected behavior**
Make is possible without any error.

**Screenshots**

**MediaElch Version:**
- GitHub actual code

**Operating System:**
 - [ ] Windows
 - [ ] macOS
 - [X] Linux (distribution: Ubuntu 20.04)

**Additional context**
Looks like the last change on "AEBN.cpp" or "AEBN.h" has caused this issue.

--- a/src/data/Locale.cpp
+++ b/src/data/Locale.cpp
@@ -6,6 +6,10 @@ namespace mediaelch {
 
 Locale Locale::English = Locale("en-US");
 
+Locale::Locale() : m_lang("en"), m_country("US")
+{
+}
+
 Locale::Locale(const QString& locale)
 {
     QStringList split = locale.split('-');

--- a/src/data/Locale.h
+++ b/src/data/Locale.h
@@ -18,6 +18,9 @@ public:
     static Locale English;
 
 public:
+    /// \brief Default constructor required for Qt containers like QVector.
+    ///        Defaults to en-US.
+    Locale();
     /*implicit*/ Locale(const char* locale) : Locale(QString(locale)) {}
     /*implicit*/ Locale(const QString& locale);
 

--- a/src/scrapers/movie/TMDb.cpp
+++ b/src/scrapers/movie/TMDb.cpp
@@ -1036,10 +1036,10 @@ void TMDb::parseAndAssignInfos(QString json, Movie* movie, QSet<MovieScraperInfo
             }
         }
 
-        if (m_locale.country() == QLocale::UnitedStates && us.isValid()) {
+        if (m_locale.country() == "US" && us.isValid()) {
             movie->setCertification(helper::mapCertification(us));
 
-        } else if (m_locale.language() == QLocale::English && gb.isValid()) {
+        } else if (m_locale.language() == "en" && gb.isValid()) {
             movie->setCertification(helper::mapCertification(gb));
 
         } else if (locale.isValid()) {

